src/terrain/elevation.js

```diff
@@ -25,7 +25,10 @@ export class Elevation {
      * point elevation, returns `defaultIfNotLoaded`.
      * Doesn't invoke network request to fetch the data.
      */
-    getAtPoint(point: MercatorCoordinate, defaultIfNotLoaded: number = 0): number {
+    getAtPoint(point: MercatorCoordinate, defaultIfNotLoaded: ?number): number | null {
+        // Force a cast to null for both null and undefined
+        if (defaultIfNotLoaded == null) defaultIfNotLoaded = null;
```

Comment:
Is `NaN` a possibility? I can imagine this API being called a lot, and NaN has the advantage that it's the same data type as valid numeric outputs, which can play well with js optimization. To be clear though, even if it might be nicer on some subjective level, though I'm not convinced this would add up to a meaningful difference in the real world.

Reply:
Ah yea, the internal API method can totally do that. what do you think would be easier for users for the public facing part? Since this is based on a fixed location is unlikely users will call this frequently unless they're moving something on the map.

It seems to me like `NaN` would mostly have the same usability as `null`, since it is also falsy and can be cast to a default with `|| 0`. But its a bit obscure bit obscure for beginners.

Reply:
Oh but also this
<img width="334" alt="Screen Shot 2021-04-19 at 3 13 30 PM" src="https://user-images.githubusercontent.com/1449257/115309946-d4bbc400-a121-11eb-9fe5-52607101697f.png">

if a suer tries to directly take this output and pass to threejs/otherwebgl library the conversion to native types is more user friendly with null

Reply:
Yeah, usability-wise, null is probably best. 👍

I thought maybe people would try to evaluate the elevation of a path (which problematically depends on loading/resolution of data), but you're right that this is probably not something you call *too* much. I'm just recalling that it you e.g. initialize object properties and worry about performance, `this.count = null` can be a meaningful performance hit compared to `this.count = NaN` since the former results in the object signature changing the type of its properties.

Reply:
Haha the cast-to-zero behavior is an awful accident, but you're right: it's friendlier. 🙈